javascript-light-bulb/main.js

```diff
@@ -0,0 +1,19 @@
+var isOn = false;
+
+function circle(event) {
+
+  if (isOn) {
+    $button.className = 'off';
+    $body.className = 'body-dark';
+
+  } else {
+    isOn = true;
+    $button.className = 'body-light on';
+    $body.className = 'body-light';
+  }
```

Comment:
This looks like it will work if you turn the light on and off once, but not after that. You should be able to turn the light on and off repetitively.